public/index.css

```diff
@@ -3,6 +3,41 @@
   --human-pink: #fa759e;
 }
 
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
```

Comment:
Did you see any difference without these attributes? Because we have one CSS file at the moment, I want it to be as short and tidy as possible. I believe we reset these attributes in [normalize.css](https://github.com/morehumaninternet/roar-server/blob/main/public/normalize.css)